configure.sh

```diff
@@ -2,4 +2,7 @@ sudo add-apt-repository -y ppa:jonathonf/python-3.6
 sudo apt-get -y update
```

Comment:
`sudo apt-get install build-essential` may be needed after this `update`, so that the system does not lack essential build tools.